Gemfile.lock

```diff
@@ -261,6 +261,7 @@ PLATFORMS
 DEPENDENCIES
   github-pages
   jekyll
+  jekyll-redirect-from
```

Comment:
@timjohnson-valtech I don't think you need to list this as it's part of `GitHub-pages` already, see https://pages.github.com/versions/